ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate

```diff
@@ -48,10 +48,6 @@
     <FullClassName>Microsoft.VisualStudio.WinRT.TemplateWizards.CreateProjectCertificate.Wizard</FullClassName>
   </WizardExtension>
   <WizardExtension>
-    <Assembly>Microsoft.VisualStudio.WinRT.TemplateWizards, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Assembly>
-    <FullClassName>Microsoft.VisualStudio.WinRT.TemplateWizards.ApplicationInsights.Wizard</FullClassName>
-  </WizardExtension>
-  <WizardExtension>
```

Comment:
I see... we had this one extra `ApplicationInsights` wizard is all.  Makes sense now.